panel/widgets/debugger.py

```diff
@@ -313,7 +313,15 @@ def update_log_counts(self, event):
 
         self.title = ', '.join(title)
 
-    def acknowledge_errors(self,event):
+    def acknowledge_errors(self, event):
         self._number_of_errors = 0
         self._number_of_warnings = 0
         self._number_of_infos = 0
+
+    @param.depends("level", watch=True)
+    def _update_level(self):
+        stream_handler = logging.StreamHandler(self.terminal)
+        stream_handler.setLevel(self.level)
+        for logger_name in self.logger_names:
+            logger = logging.getLogger(logger_name)
+            logger.addHandler(stream_handler)
```

Comment:
Won't this accumulate additional loggers every time the level changes? Can we not simply store the original `StreamHandler` and then update it with `.setLevel`?

Reply:
:man_facepalming: that is a much cleaner solution.  I will make the change now.

Reply:
Thanks, appreciate it!